node/malus/src/variants/common.rs

```diff
@@ -352,8 +373,22 @@ where
 						pov,
 						timeout,
 						response_sender,
+						commitments_hash,
 					),
 			} => {
+				// Behave normally if the `PoV` is not known to be malicious.
+				if pov.block_data.0.as_slice() != MALICIOUS_POV {
```

Comment:
I talked to @pepoviola and we want this to be determined by the backchannel rather than the PoV itself. But that's for a follow up PR :) the initial goal is achieve without it.

Reply:
Yes, the plan is to use the backchannel long term, this is just the first step